src/main/java/seedu/address/logic/commands/UnassignTaskCommand.java

```diff
@@ -0,0 +1,105 @@
+import seedu.address.model.employee.Employee;
+import seedu.address.model.project.Project;
+import seedu.address.model.task.Task;
+import seedu.address.model.task.TaskList;
+
+/**
+ * Unassigns a task in a project from the currently assigned employee.
+ */
+public class UnassignTaskCommand extends Command {
+    public static final String COMMAND_WORD = "unassignT";
+    public static final String MESSAGE_SUCCESS = "Unassigned task: %1$s from %2$s";
+    public static final String MESSAGE_USAGE =
+            COMMAND_WORD + ": Unassigns a task in a project from an employee in the project.\n"
+                    + "Parameters: "
+                    + PREFIX_PROJECT + "PROJECT_INDEX "
+                    + PREFIX_TAG + "TASK_INDEX\n"
+                    + "Example: " + COMMAND_WORD + " " + PREFIX_PROJECT + "1 " + PREFIX_TAG + "2 ";
+
+    private final Index projectIndex;
+    private final Index taskIndex;
+
+    /**
+     * Constructs an unassignTaskCommand
+     * @param projectIndex The index of the project containing the employees and tasks to be unassigned.
+     * @param taskIndex The index of the task in the project's list of tasks.
+     */
+    public UnassignTaskCommand(Index projectIndex, Index taskIndex) {
+        requireAllNonNull(projectIndex, taskIndex);
+        this.projectIndex = projectIndex;
+        this.taskIndex = taskIndex;
+    }
+
+    @Override
+    public CommandResult execute(Model model) throws CommandException {
+        requireNonNull(model);
+
+        List<Project> lastShownProjectList = model.getFilteredProjectList();
+
+        if (projectIndex.getZeroBased() >= lastShownProjectList.size()) {
+            throw new CommandException(Messages.MESSAGE_INVALID_PROJECT_DISPLAYED_INDEX);
+        }
+
+        Project projectToEdit = lastShownProjectList.get(projectIndex.getZeroBased());
+        TaskList projectTaskListToEdit = projectToEdit.getTasks();
+
+        if (taskIndex.getZeroBased() >= projectTaskListToEdit.getSize()) {
+            throw new CommandException(Messages.MESSAGE_INVALID_TASK_DISPLAYED_INDEX);
+        }
+
+        Task taskToEdit = projectTaskListToEdit.getTask(taskIndex);
+        Task editedTask = new Task(taskToEdit.getName(), taskToEdit.getDeadline(), taskToEdit.isDone());
+        TaskList editedTaskList = new TaskList();
+        editedTaskList.setTasks(projectTaskListToEdit.asUnmodifiableObservableList());
+        editedTaskList.setTask(taskIndex, editedTask);
+        Project editedProject = new Project(projectToEdit.getNameString(),
+                projectToEdit.getEmployees(),
+                editedTaskList,
+                projectToEdit.getProjectPriority(),
+                projectToEdit.getDeadline(),
+                projectToEdit.getCompletionStatus());
+        model.setProject(projectToEdit, editedProject);
+        model.updateFilteredProjectList(Model.PREDICATE_SHOW_ALL_PROJECTS);
+        Employee assignedEmployee = taskToEdit.getEmployee().get(0);
```

Comment:
if there is no `isEmpty` check for the employee list, there will be an uncaught `IndexOutOfBoundsException`. I will provide more details in my overall PR review summary comment.